src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,32 @@
+$card-width: 200px;
+$card-height: 406px;
+
+$card-container-margin: 50px;
+$card-container-width: (4 * $card-width + 3 * $card-container-margin);
+
+$button-height: 40px;
+$button-width: 166px;
+$stars-size: 16px;
+
+$main-font-family: "Roboto", sans-serif, arial;
+
+$font-size-code-review: 10px;
```

Comment:
the values you are using for more 2 times for that only it is advisable to create variable
please remove all unnecessary variables